### ISSUE TITLE
Images UI broken after switching container engine on M1

### ISSUE THREAD

**Issue by IsaSih** [rancher-sandbox/rancher-desktop#3050]
### Actual Behavior

After switching from moby to containerd on Mac M1, UI of Images tab seems to be broken and tray menu is displaying the previous engine.

### Steps to Reproduce

1. Launch Rancher Desktop
2. Select moby as container engine
3. Run a container `docker run --privileged -d --restart=no -p 8080:80 -p 8443:443 rancher/rancher`
4. Check if container is up
5. Switch the engine to containerd
6. Go to images and check if they were updated by selecting different namespaces
7. Check the container engine displayed in the tray menu
8. Check the container engine displayed in Preferences Window

### Result

UI of Images tab is broken after switching the container engines without  performing any actions with nerdctl. Tray menu is displaying the previous engine.

### Expected Behavior

User should be able to switch between container engines without any UI issues

### Rancher Desktop Version

1.5.0-701-g2d00a573

### Rancher Desktop K8s Version

1.25.0

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

macOS

### Operating System / Build Version

maOs Monterey 

### What CPU architecture are you using?

arm64 (Apple Silicon)

**Comment by rak-phillip** [comment]
I was able to reproduce this on Windows so this look to be an issue on just M1. Looks easy to reproduce by switching from moby to containerd.

**Comment by jandubois** [comment]
> I was able to reproduce this on Windows so this look to be an issue on just M1.

This is somewhat self-contradictory: If you can reproduce it on Windows, then it does not just affect M1, no?

I was just wondering if I should put additional labels on it, and it isn't clear to me if you repro'ed on Windows or not.

**Comment by rak-phillip** [comment]
> > I was able to reproduce this on Windows so this look to be an issue on just M1.
> 
> This is somewhat self-contradictory: If you can reproduce it on Windows, then it does not just affect M1, no?
> 
> I was just wondering if I should put additional labels on it, and it isn't clear to me if you repro'ed on Windows or not.

Sorry, typo in my original post that has since been fixed...

I renamed the issue previously to remove the M1 specification. I don't know if we have an "All" label that will help to clarify so I added platform labels to hopefully make it more clear that this applies to all environment.

**Comment by jandubois** [comment]
> I don't know if we have an "All" label

No, we don't. We only apply a platform label when the issue doesn't apply to all platforms.

> so this look to be an issue on just M1.

You wrote this *after* you removed the M1 from the subject, which had me confused. I guess there should be a "not" in there somewhere 😄 

Anyways, all good now.
